user_guide_src/source/helpers/date_helper.rst

```diff
@@ -27,17 +29,18 @@ The following functions are available:
     :returns:    UNIX timestamp
     :rtype:    int
 
-    Returns the current time as a UNIX timestamp, referenced either to your server's
-    local time or any PHP supported timezone, based on the "time reference" setting
-    in your config file. If you do not intend to set your master time reference to
-    any other PHP supported timezone (which you'll typically do if you run a site
-    that lets each user set their own timezone settings) there is no benefit to using
-    this function over PHP's ``time()`` function.
+    .. note:: It is recommended to use the :doc:`Time <../libraries/time>` class instead. Use ``Time::now()->getTimestamp()`` to get the current UNIX timestamp.
+
+    If a timezone is not provided, it will return the current UNIX timestamp by ``time()``.
 
     .. literalinclude:: date_helper/002.php
 
-    If a timezone is not provided, it will return ``time()`` based on the
-    **time_reference** setting.
+    If any PHP supported timezone is provided, it will return a timestamp that is off by the time difference. It is not the same as the current UNIX timestamp.
```

Comment:
```suggestion
    If any PHP supported timezone is provided, it will return a timestamp that is offset by the time difference. It is not the same as the current UNIX timestamp.
```